storage_service/locations/models/space.py

```diff
@@ -461,7 +463,11 @@ def move_rsync(self, source, destination, try_mv_local=False):
         command = ['rsync', '-t', '-O', '--protect-args', '-vv', '--chmod=ugo+rw', '-r', source, destination]
         LOGGER.info("rsync command: %s", command)
 
-        p = subprocess.Popen(command, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
+        if assume_rsync_daemon:
+            p = subprocess.Popen(command, stdout=subprocess.PIPE, stderr=subprocess.STDOUT, env={'RSYNC_PASSWORD': rsync_password})
+        else:
+            p = subprocess.Popen(command, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
```

Comment:
I better way to write this could be:

```python
kwargs = {'stdout': subprocess.PIPE, 'stderr': subprocess.STDOUT}
if assume_rsync_daemon:
    kwargs['env'] = {'RSYNC_PASSWORD': rsync_password}
p = subprocess.Popen(command, **kwargs)
```

I almost prefer this one because it's shorter, but sometimes shorter is not better!

```python
env = {'RSYNC_PASSWORD': rsync_password} if assume_rsync_daemon else None
p = subprocess.Popen(command, stdout=subprocess.PIPE, stderr=subprocess.STDOUT, env=env)
```